File: Ruby/rankB_Up/FortuneTelling/step1.rb
# 次のような「ユーザー」と「ユーザーに対応する血液型」を連想配列（辞書）として定義して、それらのキーと値のペアを順に出力してください。

hash = { Kyoko:	:B, Rio: :O, Tsubame: :AB, KurodaSensei: :A	, NekoSensei: :A }
hash.each do | name, blood|
    puts name.to_s + " " + blood.to_s
end

# 回答例2
user2blood = {'Kyoko' => 'B', 'Rio' => 'O', 'Tsubame' => 'AB', 'KurodaSensei' => 'A', 'NekoSensei' => 'A'}
user2blood.each do |user, blood|
  puts "#{user} #{blood}"
end

File: Python/rankB_Up/fortune/step1.py
# 次のような「ユーザー」と「ユーザーに対応する血液型」を連想配列（辞書）として定義して、それらのキーと値のペアを順に出力してください。


# ユーザー	ユーザーに対応する血液型
# Kyoko	B	
# Rio	O	
# Tsubame	AB	
# KurodaSensei	A	
# NekoSensei	A	



dic = {"Kyoko":"B", "Rio":"O", "Tsubame":"AB", "KurodaSensei":"A", "NekoSensei":"A"}

for key,value in dic.items():
    print(key,value)